routes/entryRoutes.js

```diff
@@ -2,21 +2,17 @@ const express = require('express');
 const router = express.Router();
 
 const {
-  addToFavorites,
   createEntry,
   deleteEntry,
   editEntry,
   getAllEntries,
   getEntry,
-  getFavorites,
 } = require('../controllers/entryController');
 
+// localhost:5000/api/v1/entries
 router.route('/').post(createEntry).get(getAllEntries);
-router
-  .route('/:id')
-  .get(getEntry)
-  .delete(deleteEntry)
-  .patch(editEntry)
-  .patch(addToFavorites);
+
+// localhost:5000/api/v1/entries/:id
+router.route('/:id').get(getEntry).delete(deleteEntry).patch(editEntry);
```

Comment:
you left out add to favourites???

Reply:
No, it's handled under editEntry. Patch was used so it will edit only the attributes passed to it.

Reply:
Yeah, got it. That's ok